File: euler1.rb
# http://projecteuler.net/problem=1
start_time = Time.now

accumulator = 0
(1..999).each do |number|
  accumulator += number if number % 3 == 0 || number % 5 == 0
end

puts accumulator
puts ((Time.now - start_time).to_f).to_s + "s"

File: euler1.swift
var counter = 1
var accumulator = 0

while counter < 1000 {
    if counter % 3 == 0 {
        accumulator += counter
    } else if counter % 5 == 0 {
        accumulator += counter
    } else {
        
    }
    counter++
}

println((\accumulator))
